tests/test_dynamodb.py

```diff
@@ -19,6 +26,112 @@ def test_dynamodb_add_doi_item_to_database(dynamodb_class):
     assert add_response["ResponseMetadata"]["HTTPStatusCode"] == 200
 
 
+@mock_dynamodb2
+def test_check_read_permissions_success(
+    dynamodb_class,
+):
+    dynamodb_class.client.create_table(
+        TableName="test_dois",
+        KeySchema=[
+            {"AttributeName": "doi", "KeyType": "HASH"},
+        ],
+        AttributeDefinitions=[
+            {"AttributeName": "doi", "AttributeType": "S"},
+        ],
+    )
+    dynamodb_class.client.describe_table(TableName="test_dois")
+    result = dynamodb_class.check_read_permissions("test_dois")
+    assert result == "Read permissions confirmed for table: test_dois"
+
+
+# While this function works as expected against stage AWS, the type of errors
```

Comment:
Could you move this to a gist for a reference instead of the commented out code here?